packages/components/fields/password-field/src/password-field.spec.js

```diff
@@ -141,13 +141,27 @@ describe('when disabled', () => {
     const { getByLabelText } = renderPasswordField({ isDisabled: true });
     expect(getByLabelText('PasswordField')).toBeDisabled();
   });
+  it('should set the input type to password', () => {
+    const { container } = renderPasswordField({ isDisabled: true });
```

Comment:
We still need to give it some value, otherwise, the test will pass just in all cases since we are testing a particular scenario:
1. when there is value
2. when readOnly and disabled
3. when input type is text

```suggestion
    const { getByLabelText } = renderPasswordField({ value: 'foo', isDisabled: true });
```